Tests/Functional/Fixtures/JobManagerTest/syncJobsTable.sql

```diff
@@ -4,6 +4,6 @@ CREATE TABLE ongr_sync_jobs (id bigint(20) unsigned NOT NULL AUTO_INCREMENT,
   `document_id` VARCHAR(32) COLLATE utf8_unicode_ci NOT NULL,
   `update_type` smallint(1) NOT NULL DEFAULT '1' COMMENT '0-partial,1-full',
   `timestamp` DATETIME NOT NULL,
-  `status` tinyint(1) NOT NULL DEFAULT '0' COMMENT '0-new,1-done',
+  `status_default` tinyint(1) NOT NULL DEFAULT '0' COMMENT '0-new,1-done',
```

Comment:
Because of this method: https://github.com/ongr-io/ConnectionsBundle/blob/master/Sync/DiffProvider/SyncJobs/TableManager.php#L189

If there are no shops, column `status` has no postfix, otherwise it appends shop's name. I'm not sure it is meaningful. In my opinion system should create separated jobs table for every shop. What do you think?